app/Http/Controllers/UtilizationMetricsApiController.php

```diff
@@ -0,0 +1,106 @@
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace App\Http\Controllers;
+
+use Exception;
+use App\Models\User;
+use Illuminate\Http\Request;
+use AdvisingApp\Ai\Models\Prompt;
+use AdvisingApp\Form\Models\Form;
+use AdvisingApp\Task\Models\Task;
+use Illuminate\Http\JsonResponse;
+use AdvisingApp\Ai\Models\AiThread;
+use AdvisingApp\Alert\Models\Alert;
+use AdvisingApp\Ai\Models\PromptUse;
+use AdvisingApp\Survey\Models\Survey;
+use AdvisingApp\Segment\Models\Segment;
+use AdvisingApp\Campaign\Models\Campaign;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\Form\Models\FormSubmission;
+use AdvisingApp\MeetingCenter\Models\Event;
+use AdvisingApp\Report\Enums\TrackedEventType;
+use AdvisingApp\Campaign\Models\CampaignAction;
+use AdvisingApp\Survey\Models\SurveySubmission;
+use AdvisingApp\Authorization\Enums\LicenseType;
+use AdvisingApp\Report\Models\TrackedEventCount;
+use AdvisingApp\StudentDataModel\Models\Student;
+use AdvisingApp\ResourceHub\Models\ResourceHubArticle;
+
+class UtilizationMetricsApiController extends Controller
+{
+    public function __invoke(Request $request): JsonResponse
+    {
+        try {
+            return response()->json([
+                'data' => [
+                    'users' => User::count(),
+                    'ai_users' => User::whereHas('licenses', function ($query) {
+                        $query->where('type', LicenseType::ConversationalAi);
+                    })->count(),
+                    'ai_exchanges' => TrackedEventCount::where('type', TrackedEventType::AiExchange)->value('count'),
+                    'saved_ai_chats' => AiThread::whereNotNull('name')->whereNotNUll('saved_at')->count(),
+                    'saved_prompts' => Prompt::count(),
+                    'prompts_inserted' => PromptUse::count(),
+                    'retention_crm_users' => User::whereHas('licenses', function ($query) {
+                        $query->where('type', LicenseType::RetentionCrm);
+                    })->count(),
```

Comment:
This `whereHas` could be shortened by using a `whereRelation` instead.

Reply:
Done